modules/ROOT/pages/single-log-out-task.adoc

```diff
@@ -1,11 +1,11 @@
-= Configure Single Logout
+= Configuring Single Signout
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-As Administrator, you can configure single logout if you use SAML. OpenID Connect does not support single logout. Using single logout, a user or user agent can log out of an authenticated environment and ensure that both service providers and identity servers process the logout correctly.
+As Administrator, you can configure single signout if you use SAML. OpenID Connect does not support single signout. Using single signout, a user or user agent can sign out of an authenticated environment and ensure that both service providers and identity servers process the signout correctly.
 
-This procedure shows how to perform single logout and how to control where the user is redirected after signing out. Redirecting the user to a different page is often preferred. You can configure such a redirect in your PingFederate configuration. You can add a `redirect_uri` query parameter to the SLO Service URL, and Anypoint Platform routes the user there rather than to the Anypoint Platform sign-in page.
+This procedure shows how to perform single signout and how to control where the user is redirected after signing out. Redirecting the user to a different page is often preferred. You can configure such a redirect in your PingFederate configuration. You can add a `redirect_uri` query parameter to the SLO Service URL, and Anypoint Platform routes the user there rather than to the Anypoint Platform sign-in page.
```

Comment:
```suggestion
The following procedure shows how to perform single signout and control where the user is redirected after signing out. 
To redirect the user to a different page, set up the redirect in your PingFederate configuration by adding a `redirect_uri` query parameter to the SLO Service URL. Anypoint Platform routes the user to that page rather than to the Anypoint Platform sign-in page.
```